sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -212,6 +212,11 @@ protected void SendInitialized()
 			OnInitialized();
 		}
 
+		public Vector2d getGeoPositionOfWorldPoint(Vector3 realworldPoint)
+		{
+			return (Quaternion.Inverse(this.transform.rotation) * (realworldPoint - this.transform.position)).GetGeoPosition(this.CenterMercator, this.WorldRelativeScale);
```

Comment:
`this.transform` should be replaced with `_root`, in the case that a separate transform is being used as the parent.